tinymce/widgets.py

```diff
@@ -55,7 +55,7 @@ def get_mce_config(self, attrs):
         mce_config = tinymce.settings.DEFAULT_CONFIG.copy()
         if "language" not in mce_config:
             mce_config["language"] = get_language_from_django()
-        if mce_config["language"] == "en_US":
+        if mce_config["language"] in ["en_GB", "en_US"]:
```

Comment:
What about `startswith("en_")` to cover all variants?
(unless the language param can only be GB or US, not CA or all the other possibilities)